Cargo.toml

```diff
@@ -13,8 +13,8 @@ description = """
 Generates an ebuild for a package using the in-tree eclasses.
 """
 
-[badges]
-travis-ci = { repository = "cardoe/cargo-ebuild" }
+[badges.travis-ci]
+repository = "cardoe/cargo-ebuild"
```

Comment:
What's the improvement here? I see the first form in most projects.

Reply:
personal porn-idiomatic